lib/node_modules/@stdlib/utils/configdir/README.md

```diff
@@ -0,0 +1,132 @@
+# configdir
+
+> Return a directory for user-specific configuration files.
+
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var configdir = require( '@stdlib/utils/configdir' );
+```
+
+#### configdir( \[path\] )
+
+Returns a directory for user-specific configuration files.
+
+``` javascript
+var dir = configdir();
+// e.g., returns '/Users/<username>/Library/Preferences'
+```
+
+To append a `path` to the base directory, provide a `path` argument.
+
+``` javascript
+var dir = configdir( 'appname/config' );
+// e.g., returns '/Users/<username>/Library/Preferences/appname/config'
+```
+
+On non-windows platforms, if the function is unable to locate the current user's [`home`][@stdlib/utils/homedir] directory, the function returns `null`.
```

Comment:
We should probably capitalize Windows here => `non-Windows`. Same goes for similar usage in `./test/test.js`.